### DISCUSSION TITLE
Which Languages Would You Like Locker to Support?

### DISCUSSION THREAD

**Discussion by issimonsi** [lockerpm/.github#4] [Announcements]
Hey there!

We highly value the feedback and preferences of our users, and we are currently in the process of expanding language support for our application. We believe that providing Locker in multiple languages will enhance the user experience and ensure that our app reaches a broader audience.

We would love to hear your thoughts and suggestions on the languages you would like to see supported in Locker. Please take a moment to share your desired languages in the comments below. We keep a list of requested languages on this page: https://cystack.notion.site/Locker-Translation-Guide-bb4e4fc4c23d4bbc994375035b124829

📣 Just to let you know, the localization of Locker in languages (other than English and Vietnamese) will depend on crowdsourcing. This means that you have the opportunity to directly contribute to the localization process!
To participate in the localization project for any language and choose the requested languages, simply find the project link on the page above and join.

Your involvement will help make Locker more accessible in your native language and bring greater value to the community. We truly appreciate your contribution! 💚

**Comment by michaelsantosti** [comment]
Hello, I use Locker and have recently been recommending it to my friends and professional contacts. I believe that a translation to Portuguese (Brazil) would be very helpful. How can I contribute to the translation for this language?

**Reply by luongnh261** [reply]
Hey @michaelsantosti, 

Thank you for your support and interest in the Locker Translation program! 

For the Brazilian Portuguese language, we expect you to translate for the following platforms: Web app, Mobile app, and Extension. The translation volume is as follows:
- A file for web application/desktop app: about 10,000 English words
- A file for browser extension: about 3000 English words
- A file for mobile application + a file for app description on store: about 4000 English words

We offer incentives for translators with Premium subscriptions, which can take a look at this link: https://cystack.notion.site/Locker-Translation-Guide-bb4e4fc4c23d4bbc994375035b124829#e78859ae7cc64294a58f82bbdf485f1c 

To start with, please sign up for a [Crowdin](https://accounts.crowdin.com/register?continue=%2Fjoin%3F_gl%3D1%252Aknys1b%252A_up%252AMQ..%252A_ga%252AOTczNDI3NDY3LjE3MjU4NzM4MTg.%252A_ga_SRNRQ29Q3E%252AMTcyNTg3MzgxOC4xLjAuMTcyNTg3MzgxOC4wLjAuMjgwNzI3OTA3) account, then you can start the translation at the project link: https://crowdin.com/project/lockerportuguese

If you have any questions or concerns, please reach out to us anytime, or view [the translation guide](https://cystack.notion.site/Locker-Translation-Guide-bb4e4fc4c23d4bbc994375035b124829#1e5ef4ff879f43cb970c8e1617b65f1e) here!

We greatly value your contribution, which will help build a stronger Locker community. Look forward to having you onboarded soon!

P/S: For this project, we also need a proofreader to double-check the translation. Please help us reach out to other language professionals for this position if possible. Many thanks!!

**Comment by artdome** [comment]
Hi, I would be glad to help with translation to Russian or Latvian languages if necessarily, just start using the app and found it very nice with good potential.

**Reply by luongnh261** [reply]
Hey @arthurwrls,

We would always welcome new contributors to the Translation Project. For the two languages you mentioned, we think it would be great if you could do Russian first (which would be more beneficial for the community because of a larger user base). 

We expect you to translate for the following platforms: Web app, Mobile app, and Extension. The translation volume is as follows:

- A file for web application/desktop app: about 10,000 English words
- A file for browser extension: about 3000 English words
- A file for mobile application + a file for app description on store: about 4000 English words

We offer incentives for translators with Premium subscriptions, which can take a look at this link: https://cystack.notion.site/Locker-Translation-Guide-bb4e4fc4c23d4bbc994375035b124829#e78859ae7cc64294a58f82bbdf485f1c

If this works for you, please send a confirmation email about the language to be translated to social@locker.io so we can guide you further. Cheers!

**Reply by artdome** [reply]
Thanks, just sent email to social@locker.io,
Thanks
